tests/wpt/mozilla/tests/webgl/tools/import-conformance-tests.py

```diff
@@ -13,12 +13,14 @@
 PATCHES_1X = [
     ("js-test-pre.patch", "resources/js-test-pre.js"),
     ("unit.patch", "conformance/more/unit.js"),
+    ("unit2.patch", "conformance/more/unit.js"),
     ("timeout.patch", None)
 ]
 # Patches for conformance tests 2.0.x
 PATCHES_2X = [
     ("js-test-pre2.patch", "js/js-test-pre.js"),
     ("unit.patch", "conformance/more/unit.js")
+    ("unit2.patch", "conformance/more/unit.js")
```

Comment:
Should this patch just be merged into `unit.patch`?

Reply:
I don't know if that would affect the line numbers of unit.patch. This seemed easier.